### ISSUE TITLE
babel-eslint support

### ISSUE THREAD

**Issue by mysticatea** [mysticatea/vue-eslint-parser#1]
Currently, `vue-eslint-parser` is using the default parser (espree).
It needs the capability to use custom parsers to parse `<script>` element.

For example:

```json
{
    "parser": "vue-eslint-parser",
    "parserOptions": {
        "vue-eslint-parser": {
            "parser": "babel-eslint"
        },
        "sourceType": "module",
        "ecmaVersion": 2017
    }
}
```

**Comment by mysticatea** [comment]
Known limitations: it cannot support relative paths for the parser option. `vue-eslint-parser` cannot know the location of the `.eslintrc.*` file which has the option.

**Comment by mysticatea** [comment]
Closed by b16a6c1402276631b286b8e4d8e5ac2a199d3563
